system-tests/scripts/bootstrap-docker-container.sh

```diff
@@ -36,6 +36,9 @@ export CYPRESS_CACHE_FOLDER=/tmp/CYPRESS_CACHE_FOLDER/
 export npm_config_cache=/tmp/npm_config_cache/
 export npm_config_package_lock=false
 
+mkdir /tmp/npm_config_cache
```

Comment:
I had to add this, none of our existing binary system tests seem to actually install any node modules. These ones do, and without this line I get some random permission error when doing npm install in Docker.

Reply:
Good find, I remember running in to that permission error while setting this up and having NFC why it was happening.